scripts/apps/ingest/directives/IngestUserDashboard.ts

```diff
@@ -27,9 +27,17 @@ export function IngestUserDashboard(api, userList, privileges, moment) {
                     },
                 };
 
-                api.ingest.query(criteria).then((result) => {
-                    scope.ingested_count = result._meta.total;
-                });
+                if (scope.item.content_types.includes('event') || scope.item.content_types.includes('planning')) {
+                    const param = scope.item.content_types.includes('event') ? 'events' : 'planning';
```

Comment:
not sure if param is a best name, something like resource would be more descriptive